app.js

```diff
@@ -0,0 +1,16 @@
+
+let cardFront = document.querySelectorAll('.card-front');
+
+let cardBack = document.querySelectorAll('.card-back');
+let cardArray = [...cardBack]
+// REveal pokemon on click
+for(let i=0; i<cardArray.length; i++){
+    
+        cardArray[i].addEventListener('click',()=>{
+            for(let j = 0; j<cardFront.length; j++){
+                if(i == j){
+                    cardFront[j].style.transform = 'none';

```

Comment:
As for looping the card, as @Dex-theDev suggested, you can look into using [forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) method to loop through a nodeList.

```suggestion
let cards = document.querySelectorAll('.card');

cards.forEach(card => {
  // Do something
})
```